index.html

```diff
@@ -769,21 +769,26 @@ <h2 id='ldpcvpatch'>HTTP PATCH</h2>
         <section>
           <h2 id='ldpcvpost'>HTTP POST</h2>
           <p>
-            If an <a>LDPCv</a> supports <code>POST</code>, <code>POST</code> SHOULD be understood to create a new
-            <a>LDPRm</a> contained by the <a>LDPCv</a>, reflecting the state of the <a>LDPRv</a> at the time of the
-            <code>POST</code>. If supported, but the representation at the time of version creation can only be that
-            which is current for the <a>LDPRv</a>, the <code>Accept-Post</code> header of any response from the
-            <a>LDPCv</a> SHOULD indicate that no request body is accepted via the form <code>Accept-Post: */*;
-            p=0.0</code>, and that implementation SHOULD respond to any body-containing <code>POST</code> to that
-            <a>LDPCv</a> with a 415 response and a link to an appropriate constraints document (see
-            <a href='https://www.w3.org/TR/ldp/#ldpr-gen-pubclireqs'>LDP 4.2.1.6</a>). As per [[!LDP]], any resource
-            created via <code>POST</code>, in this case an <a>LDPRm</a>) SHOULD be advertised in the response's
-            <code>Location</code> header.
+            If an <a>LDPCv</a> supports <code>POST</code>, a <code>POST</code> that does not contain a
+            <code>Memento-Datetime</code> header SHOULD be understood to create a new <a>LDPRm</a> contained by the
+            <a>LDPCv</a>, reflecting the state of the <a>LDPRv</a> at the time of the <code>POST</code>.
+            Implementations MUST fail such requests that also contain a non-empty body by responding
+            with a 400 Bad Request.
```

Comment:
I'm not sure I agree with the non-empty body. If it's a dumb implementation, that may be how the versions are created. If the impl doesn't support request entities on version creation via POST, that should be indicated somehow (as in the next 2 paragraphs).

Reply:
That is to say that a combination of `Accept:` from the server and a 400 response should tell the client that the operation was illegal - I don't think we should specify anything else here.